### DISCUSSION TITLE
What is REST?

### DISCUSSION THREAD

**Discussion by giscus** [PradeepLoganathan/pradeepl-blog#10] [BlogPosts]
# What is REST?

The Web has radically transformed the way we produce and share information. It has democratized information sharing and has revolutionized how information is created and shared. It has also been through major inflection points named only as software engineers would.
Web 1.0 – Was about static pages.
Web 2.0 – Is/was about responsive, dynamic content targeted for various form factors such as mobile and tablets. It has evolved primarily into a “Mobile First” movement.

http://localhost:35205/blog/rest/what-is-rest/

**Comment by PradeepLoganathan** [comment]
sdfsdfsdf
